draft-ietf-rats-architecture.md

```diff
@@ -85,33 +85,23 @@ remote attestation procedures (RATS).
 
 # Introduction
 
-In Remote Attestation Procedures (RATS), one peer (the "Attester")
-produces believable information about itself - Evidence - to enable
-a remote peer (the "Relying Party") to decide whether to consider that
-Attester a trustworthy peer or not.
-RATS are facilitated by an additional vital party, the Verifier. The
-Verifier appraises Evidence via Appraisal Policies and creates the
-Attestation Results to support Relying Parties in their decision
-process.
-
-This documents defines a flexible architecture with corresponding roles
-and their interaction via conceptual messages. Additionally, this document
-defines a universal set of terms that can be mapped to various existing
-and emerging Remote Attestation Procedures. Common role compositions and
-data flows, such as the "Passport Model" and the "Background-Check Model"
-are illustrated to enable readers of this document to map their current
-and emerging solutions to the architecture provided and the corresponding
-terminology defined. A common terminology that provides a well-understood semantic meaning to
+In Remote Attestation Procedures (RATS), one peer (the "Attester") produces believable information about itself - Evidence - to enable a remote peer (the "Relying Party") to decide whether to consider that Attester a trustworthy peer or not.
```

Comment:
This is hard to review since the line breaks were changed to combine many lines into few lines.
Can you revert the line break changes and keep source lines to a short length like before, so
github diffs are easier?